index.html

```diff
@@ -115,72 +152,111 @@ <h2 class="title">About Me</h2>
-            </div>
-            <div>
-                <textarea name="description" id="description"  rows="3" cols="41" maxlength="500" placeholder="Enter text here" required></textarea>
-            </div>
-            <div>
-                <button class="submit-button" type="submit" >Get in touch</button>
+            <div class="contact__container">
+                <h2 class="title">I'm always interested in hearing about new projects, so if you'd like to chat please
+                    get
+                    in touch.</h2>
+                <form action="https://formspree.io/f/myyowwdj" method="POST" class="contact-form form-mobile">
+                    <div>
+                        <div class="full__name">
+                            <input type="text" name="full-name" id="full-name" size="40" maxlength="30"
+                                placeholder="Full Name" required>
+                        </div>
+                        <div class="email">
+                            <input type="email" name="email" id="email" size="40" maxlength="30" placeholder="Email"
+                                required>
+                        </div>
+                        <textarea name="description" id="description" rows="3" cols="41" maxlength="500"
+                            placeholder="Enter text here" required></textarea>
+                    </div>
+                    <div>
+                        <button class="submit-button" type="submit">Get in touch</button>
+                    </div>
+                </form>
+                <form action="https://formspree.io/f/myyowwdj" method="POST" class="contact-form form-desktop">
+                    <div class="contact__first__last_name">
+                        <div>
+                            <input type="text" name="first-name" id="first-name" size="40" maxlength="30"
+                                placeholder="First Name" required>
+                        </div>
+                        <div>
+                            <input type="text" name="last-name" id="last-name" size="40" maxlength="30"
+                                placeholder="Last Name" required>
+                        </div>
+                    </div>
+
+                    <div>
+                        <div class="email">
+                            <input type="email" name="email" id="email-desktop" size="40" maxlength="30" placeholder="Email"
+                                required>
+                        </div>
+                        <textarea name="description" id="description-desktop" rows="3" cols="41" maxlength="500"
+                            placeholder="Enter text here" required></textarea>
+                    </div>
+                    <div>
+                        <button class="submit-button" type="submit">Get in touch</button>
+                    </div>
+                </form>

```

Comment:
- [ ] Please, I would suggest you avoid repeating yourself, this piece of code right here is meaningless since you have already created a form above, please use that same one, (as for the mobile version), there must be only one input field for the full name

- [ ] NOT THIS 

<img width="570" alt="frm" src="https://user-images.githubusercontent.com/79058364/154720945-71e2019c-7799-45cb-b958-dce899083ff4.png">

- [ ] BUT THIS INSTEAD, FOR MOBILE AND DESKTOP

<img width="351" alt="frmcrev" src="https://user-images.githubusercontent.com/79058364/154721049-0135f82b-970b-4584-b557-8f484c5493de.png">